test/unit_test_create_dataset.py

```diff
@@ -30,9 +32,39 @@ def test_files(self):
                 truth_dict = json.load(truth_file)
         ddiff = DeepDiff(output_dict, truth_dict, ignore_order=True)
 
+        # if there are differences
         self.assertTrue(len(ddiff) == 0,
-                        "Error: The created json is not equal to truth file")
+                        "Error: The created json is not equal to truth file\n{}".format(ddiff))
 
+    def test_validate_table_and_column(self):
+        current_file_dir = os.path.dirname(__file__)
+        input_tables = create_dataset.get_json(os.path.join(current_file_dir, "input_files/tables"))
+
+        # check an expected valid table and column pair
+        self.assertTrue(create_dataset.validate_table_and_column(table_name="table_2",
+                                                                 column_name="column_1",
+                                                                 input_tables=input_tables),
+                        "Error: The valid table and column name could not be validated using input JSON tables")
+
+        # check an expected invalid table name
+        with self.assertRaises(SystemExit) as cm:
+            create_dataset.validate_table_and_column(table_name="invalid_table_name",
+                                                     column_name="column_1",
+                                                     input_tables=input_tables)
+        the_exception = cm.exception
+        self.assertEqual(the_exception.code,
+                         errno.EBADF,
+                         "Error: The invalid table name was validated using input JSON tables")
+
+        # check an expected invalid column name
```

Comment:
Can we also check the case when a column references multiple datatypes, and assert that it fails validation too?